inappmessaging/USERGUIDE.md

```diff
@@ -41,20 +42,22 @@ Launches the URI in an external  browser on tap.
 Launches the URI in an external  browser on tap.
 
 ### Triggers
-This are events sent from the host app which are used as conditions if a specific campaign/message should be displayed.
+These are events sent from the host app that are used as conditions if a specific campaign/message should be displayed.
 
-#### #1 App Start Event (pre-defined)
-Event which the host app can send after every app launch, including first launch, or resume.
+#### #1 `AppStartEvent` (pre-defined)
+Event that the host app can log after app launch from terminated stated (e.g. Main Activity's `Activity#onStart()`).
 
-#### #2 Login Successful Event (pre-defined)
-Event which host app can send after every time user login.
+App Start Event is persistent, meaning, once it's logged it will always satisfy corresponding trigger in a campaign. All subsequent logs of this event are ignored. Campaigns that require only AppStartEvent are shown once per app session.
 
-#### #3 Purchase Successful Event (pre-defined)
-Event which the host app can send after every purchase.
+#### #2 `LoginSuccessfulEvent` (pre-defined)
+Event that the host app can log every time user logs in successfully.
 
-#### #4 Custom Event (customizable)
-Event which host app can send after app-defined states or conditions.
-Custom events can have attributes with names and values. Attributes can be integer, double, String, boolean, or java.util.Date type.
+#### #3 `PurchaseSuccessfulEvent` (pre-defined)
+Event that the host app can log after every successful purchase.
+
+#### #4 `CustomEvent` (customizable)
+Event that host app can send after app-defined states or conditions.
```

Comment:
I don't understand
"after app-defined states or conditions"

Is it after app-defined states are reached or conditions are met"?

Reply:
Yes, that is correct. Updated to suggestion.